tests/conftest.py

```diff
@@ -258,33 +258,15 @@ def tmp_sqlite(tmpdir_factory):
 
 @pytest.fixture(scope="function", autouse=True)
 def clean_repository():
-    if Config.global_config.repository_type == "sql":
-        close_sql_database_session_connection()
-        init_managers()
-    init_config()
-    init_scheduler()
-    init_managers()
-    init_config()
-
-    yield
+    from sqlalchemy.orm import close_all_sessions
 
-    if Config.global_config.repository_type == "sql":
-        close_sql_database_session_connection()
-        init_managers()
+    close_all_sessions()
     init_config()
     init_scheduler()
     init_managers()
     init_config()
 
-
-def close_sql_database_session_connection():
-    _ScenarioRepositoryFactory._build_repository().repo.session.close_all()
-    _PipelineRepositoryFactory._build_repository().repo.session.close_all()
-    _DataRepositoryFactory._build_repository().repo.session.close_all()
-    _TaskRepositoryFactory._build_repository().repo.session.close_all()
-    _JobRepositoryFactory._build_repository().repo.session.close_all()
-    _CycleRepositoryFactory._build_repository().repo.session.close_all()
-    _VersionRepositoryFactory._build_repository().repo.session.close_all()
+    yield
```

Comment:
```suggestion
    yield
    
    close_all_sessions()
    
    init_config()
    init_scheduler()
    init_managers()
    init_config()
```
Previously, we have one block of **clean everything** before `yield` and one after `yield`.

Do we need both?